.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
       - id: black
         language_version: python3
         exclude: ^alembic/versions/
-  -   repo: https://gitlab.com/pycqa/flake8
```

Comment:
I know this should not be in here. I got weird auth errors when downloading from gitlab. I suggest to just use the github mirror. If requested, I can break this out into another PR

Reply:
I'll give this a try, hope you don't mind pushing to your PR. If things are green I'll merge this in.